ttnn/cpp/ttnn/tensor/tensor.cpp

```diff
@@ -625,17 +655,23 @@ Tensor Tensor::pad(const Shape& output_tensor_shape, const Shape& input_tensor_s
     const auto padding = Padding(dimensions_pads, Padding::PadValue::Any);
     auto output_shape_with_padding = Shape(output_tensor_shape, padding);
 
-    return tensor_impl::pad_wrapper(*this, output_shape_with_padding, input_tensor_start, pad_value);
+    auto output = tensor_impl::pad_wrapper(*this, output_shape_with_padding, input_tensor_start, pad_value);
+    GraphTracker::instance().track_end_op(output);
+    return output;
 }
 
 Tensor Tensor::unpad(const Shape& output_tensor_start, const Shape& output_tensor_end) const {
     ZoneScoped;
+    GraphTracker::instance().track_begin_op("Tensor::unpad", output_tensor_start, output_tensor_end);
```

Comment:
why not do something like this instead of the literals:
https://godbolt.org/z/ennqza145

Reply:
@arakhmati looks like a good idea.